npm/builder.js

```diff
@@ -134,80 +135,70 @@ Builder.prototype.generateUMD = function (callback) {
 
   async.series([
-    function (callback) { builder.build(callback); },
-    function (callback) { builder.runTest(callback); },
-    function (callback) { builder.copyToDist(callback); }
-  ], function () {
-    log.success('Done in ' + process.hrtime(start)[0] + 's');
-  });
+    callback => builder.build(callback),
+    callback => builder.runTest(callback),
+    callback => builder.copyToDist(callback)
+  ], () => log.success(`Done in ${process.hrtime(start)[0]} s`));
 };
 
 Builder.prototype.prepareRelease = function (releaseVersion, callback) {
-  log.task('Release version: ' + releaseVersion);
+  log.task(`Release version: ${releaseVersion}`);
 
   if (process.env.DRY_RUN) {
     log.warn('Dry run! To perform the release, run the command again without DRY_RUN environment variable');
   } else {
-    bfs.updateFileSync('package.json', /"version": "(.*?)"/g, '"version": "' + releaseVersion + '"');
+    this.execSync(`npm version ${releaseVersion}`);
   }
   callback();
 };
 
-Builder.prototype.commit = function (releaseVersion, callback) {
-  this.execSync('git add -A .');
-  this.execSync('git commit -m "Release ' + releaseVersion + '"');
-  this.execSync('git tag v' + releaseVersion);
-  callback();
-};
-
-Builder.prototype.prepareNextIteration = function (callback) {
-  this.removeDistDirSync();
-  this.execSync('git add -A .');
-  this.execSync('git commit -m "Prepare for next development iteration"');
-  callback();
-};
-
 Builder.prototype.runTest = function (callback) {
   this.execSync('npm run test');
   callback();
 };
 
-Builder.prototype.publish = function (callback) {
-  if (process.env.SKIP_PUBLISH) {
-    log.info('SKIP_PUBLISH environment variable is defined, skipping "publish" task');
-    callback();
-    return;
+Builder.prototype.pushRelease = function () {
+  const remoteName = child_process.execSync('git remote -v').toString('utf8')
+    .split(/\r?\n/)
+    .filter(line => line.includes('(push)') && line.includes('asciidoctor/asciidoctor.js.git'))
+    .map(line => line.split('\t')[0])
+    .reduce((a, b) => a + b, '');
+
+  if (remoteName) {
+    this.execSync(`git push ${remoteName} master`);
+    this.execSync(`git push ${remoteName} --tags`);
+    return true;
+  } else {
+    log.warn('Unable to find the remote name of the original repository asciidoctor/asciidoctor.js');
+    return false;
   }
-  this.execSync('npm publish');
-  callback();
 };
 
-Builder.prototype.completeRelease = function (releaseVersion, callback) {
+Builder.prototype.completeRelease = function (releasePushed, callback) {
   log.info('');
   log.info('To complete the release, you need to:');
-  log.info('[ ] push changes upstream: `git push origin master && git push origin v' + releaseVersion + '`');
+  if (!releasePushed) {
+    log.info('[ ] push changes upstream: `git push origin master && git push origin --tags');
+  }
   log.info('[ ] publish a release page on GitHub: https://github.com/asciidoctor/asciidoctor.js/releases/new');
```

Comment:
This step is not necessary now until my PR #345.
GH releases are automatically created.